src/js/components/SelectMultiple/stories/SearchFetch.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Box } from 'grommet';
+import { SelectMultiple } from '../SelectMultiple';
+
+const locationList = [
```

Comment:
From offline discussion, I agree with not adding the storybook to the published set. Or if we do, build it out to be more representative of fetching from an API.

Reply:
Yeah I'm thinking we will remove this story before committing